workitem/workitemtype_repository.go

```diff
@@ -154,6 +156,18 @@ func (r *GormWorkItemTypeRepository) Create(ctx context.Context, spaceID uuid.UU
 		return nil, errors.NewInternalError(err.Error())
 	}
 
+	// create relationship between workitemtype and category
+	if categoryID != uuid.Nil {
```

Comment:
I wonder if you better use a `nil` pointer to `uuid.UUID` in order to specify a non-existing category. If you don't have a description for example, we pass in `nil` as well. So why make a difference for the category?